### ISSUE TITLE
after using this my app become slow

### ISSUE THREAD

**Issue by kelumsampath** [solaomosola/react-native-radio#1]

**Comment by solaomosola** [comment]
Can you give more information? Did the component work as planned?

**Comment by kelumsampath** [comment]
Yes, i'm using react native router flux navigation . After applying radio
button.it become slow navigate between two screens.can i know why is it?
